diff --git a/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components-upgrade.js b/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components-upgrade.js
index 55bdee8..157e353 100644
--- a/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components-upgrade.js
+++ b/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components-upgrade.js
@@ -267,9 +267,9 @@ var DashboardBridgeService = /** @class */ (function () {
      * @param {?=} componentDefinitions
      * @return {?}
      */
-    function (componentDefinitions) {
+    function (componentDefinitions, isAsync) {
         var _this = this;
-        if (componentDefinitions === void 0) { componentDefinitions = []; }
+        if (componentDefinitions == null) { componentDefinitions = []; }
         componentDefinitions.forEach((/**
          * @param {?} cmp
          * @return {?}
@@ -280,7 +280,7 @@ var DashboardBridgeService = /** @class */ (function () {
             /** @type {?} */
             var templateConfig = "<c8y-widget-config-downgrade\n        mode=\"config\"\n        componentid=\"" + cmp.id + "\"\n        [formref]=\"forms.componentForm\"\n        [config]=\"config\">\n      </c8y-widget-config-downgrade>";
             /** @type {?} */
-            var templateUrl = "widget-template-" + cmp.id + ".html";
+            var templateUrl = isAsync ? "async-widget-template.html" : "widget-template-" + cmp.id + ".html";
             /** @type {?} */
             var configTemplateUrl = cmp.configComponent ? "widget-template-config-" + cmp.id + ".html" : undefined;
             /** @type {?} */
@@ -950,24 +950,24 @@ var Ng1ActionFactoryService = /** @class */ (function () {
      * @return {?}
      */
     function (tabs) {
-        /**
-         * This function is doing the same process as function redirect in the file
-         * modules/core/ui/navigation/tabs.provider.js
-         * That function  is not run because bridge.service.ts overrides the method addTab where the redirect() was called.
-         * @type {?}
-         */
-        var redirectedTab = tabs.find((/**
-         * @param {?} tab
-         * @return {?}
-         */
-        function (tab) { return tab.redirectedTo; }));
-        var _a = __read(tabs, 1), topPriorityTab = _a[0];
-        if (redirectedTab && !topPriorityTab.redirectedTo) {
-            this.$location.replace();
-            this.$location.path(topPriorityTab.path);
-            topPriorityTab.redirectedTo = true;
-            redirectedTab.redirectedTo = false;
-        }
+        // /**
+        //  * This function is doing the same process as function redirect in the file
+        //  * modules/core/ui/navigation/tabs.provider.js
+        //  * That function  is not run because bridge.service.ts overrides the method addTab where the redirect() was called.
+        //  * @type {?}
+        //  */
+        // var redirectedTab = tabs.find((/**
+        //  * @param {?} tab
+        //  * @return {?}
+        //  */
+        // function (tab) { return tab.redirectedTo; }));
+        // var _a = __read(tabs, 1), topPriorityTab = _a[0];
+        // if (redirectedTab && !topPriorityTab.redirectedTo) {
+        //     this.$location.replace();
+        //     this.$location.path(topPriorityTab.path);
+        //     topPriorityTab.redirectedTo = true;
+        //     redirectedTab.redirectedTo = false;
+        // }
     };
     /**
      * @return {?}
@@ -1679,7 +1679,8 @@ var DashboardUpgradeModule = /** @class */ (function () {
                     declarations: [WidgetComponent],
                     entryComponents: [WidgetComponent],
                     // Uncoment the following line for demo
-                    exports: [DashboardWidgetDemoModule],
+                    // exports: [DashboardWidgetDemoModule],
+                    exports: [WidgetComponent],
                     providers: [
                         {
                             provide: DashboardBridgeService,
@@ -1791,25 +1792,25 @@ function c8yAlertDecorator($delegate, $rootScope, $injector) {
      * @return {?}
      */
     function transformAlert(alert) {
-        /** @type {?} */
-        var newAlert = __assign({}, alert);
         if (alert.onClose) {
-            newAlert.onClose = (/**
+            var onClose = alert.onClose;
+            alert.onClose = (/**
              * @return {?}
              */
             function () {
-                $injector.invoke(alert.onClose);
+                $injector.invoke(onClose);
             });
         }
         if (alert.onDetail) {
-            newAlert.onDetail = (/**
+            var onDetail = alert.onDetail;
+            alert.onDetail = (/**
              * @return {?}
              */
             function () {
-                $injector.invoke(alert.onDetail);
+                $injector.invoke(onDetail);
             });
         }
-        return newAlert;
+        return alert;
     }
     return $delegate;
 }
diff --git a/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components.js b/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components.js
index 9e86f1b..ff49b76 100644
--- a/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components.js
+++ b/node_modules/@c8y/ngx-components/fesm5/c8y-ngx-components.js
@@ -1497,7 +1497,7 @@ var NavigatorNode = /** @class */ (function () {
     function NavigatorNode(data) {
         this.children = [];
         this.parents = [];
-        this.routerLinkExact = true;
+        this.routerLinkExact = false;
         this.open = false;
         this.hidden = false;
         this.draggable = false;
@@ -10638,7 +10638,7 @@ var AppSwitcherComponent = /** @class */ (function () {
         this.ui = ui;
         this.applications = applications;
         var _a = this.ui, currentUser = _a.currentUser, currentTenant = _a.currentTenant;
-        combineLatest(currentUser, currentTenant).pipe(first()).subscribe((/**
+        combineLatest(currentUser, currentTenant).subscribe((/**
          * @param {?} __0
          * @return {?}
          */
@@ -11402,7 +11402,7 @@ var NavigatorNodeComponent = /** @class */ (function () {
             open = !this.node.open;
         }
         else {
-            open = this.routerLinkActive && this.routerLinkActive.isActive ? true : !this.node.open;
+            open = true;
         }
         if (open !== this.node.open) {
             this.expandCollapse(open, false, (_a = {}, _a[from$$1] = true, _a.$event = $event, _a));
diff --git a/node_modules/@c8y/ngx-components/upgrade/dashboard/dashboard-bridge.service.d.ts b/node_modules/@c8y/ngx-components/upgrade/dashboard/dashboard-bridge.service.d.ts
index 75e492b..ca741bb 100644
--- a/node_modules/@c8y/ngx-components/upgrade/dashboard/dashboard-bridge.service.d.ts
+++ b/node_modules/@c8y/ngx-components/upgrade/dashboard/dashboard-bridge.service.d.ts
@@ -5,5 +5,5 @@ export declare class DashboardBridgeService {
     constructor(ng1Injector: any, componentDefinitions: DynamicComponentDefinition[]);
     readonly ng1Components: any;
     readonly $templateCache: any;
-    registerComponents(componentDefinitions?: any[]): void;
+    registerComponents(componentDefinitions?: any[], isAsync?: boolean): void;
 }
